<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>emp</class>
 <widget class="QDialog" name="emp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>980</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget {	
	border:1px solid #dfdfdf;
	border-radius: 5px;
	background-color: #ffffff;
}

QPushButton{
	
	border-raduis:5px
}
#side_menu{
	border-radius: 20px;
	background-color: rgb(0,0,0);
}
#top_widget{

	background-color: rgb(0,0,0);
}
#form{
	
	
	background-color: rgb(255, 255, 255);
border-top-left-radius: 50px;
}

#widget QLineEdit{
font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgb(0,0,0);
padding-bottom:7px

}
#widget{
	
	/* background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);*/
	/*background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 0.1 #646464, stop: 1 #5d5d5d);*/
    background-color:QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4aafdb, stop: 1 #090763);
}
#options{
background-color:  rgb(0,0,0);
border-bottom-right-radius: 50px;

}
#widget #recherche_2{
	
	background-color: rgb(204, 204, 204);
	border:1px solid rgb(0,0,0);
}

#ajouter{
 color: rgb(0,0,0);
    background-color: rgb(185, 185, 185);
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 30px;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;
}


#modifier{
 color: rgb(0,0,0);
    background-color: rgb(185, 185, 185);
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 30px;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;
}

#supprimer{
  color: rgb(0,0,0);
    background-color: rgb(185, 185, 185);
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 30px;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;
}

#ajouter:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
}

#modifier:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
}

#supprimer:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
}

#top_widget #label
{
	
	font: 75 14pt &quot;Segoe Print&quot;;
}

#pdf{
	
	background-color: #071e26;
border:0px ;
}
#stat{
	
	background-color: #071e26;
border:0px ;
}
#widget_2 QLineEdit {
		background-color: rgb(204, 204, 204);
	border:1px solid rgb(0,0,0);

}
#widget_2{
	
	border-radius: 20px;
	background-color:  rgb(0,0,0);

}
QPushButton{
	background-color: rgb(204, 204, 204);
}

</string>
  </property>
  <widget class="QWidget" name="top_widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1851</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>1760</x>
      <y>20</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 160px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/retour2.png</normaloff>:/new/prefix1/icons/retour2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>20</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/emp.png</normaloff>:/new/prefix1/icons/emp.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>1651</width>
     <height>831</height>
    </rect>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>130</y>
      <width>831</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-raduis:5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="form">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>551</width>
      <height>711</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="options">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>131</width>
      <height>711</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="supprimer">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>340</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>supprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modifier">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>240</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>modifier</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nom">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>  nom</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ajouter">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>140</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>ajouter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pre">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>  prénom</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_num">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>  numéro de télèphone</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_cin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>  CIN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_mail">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>  adresse e-mail</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_add">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>  adresse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_mdp">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>300</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string> mot de passe</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>30</y>
      <width>831</width>
      <height>91</height>
     </rect>
    </property>
    <widget class="QComboBox" name="comboBox_search_by">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>50</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MV Boli&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>chercher par:</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>prenom</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_order_sort">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MV Boli&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>ordre de tri:</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>croissant</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>décroissant</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="recherche_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>  recherche</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_sort_by">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MV Boli&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>trier par:</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>prenom</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>-5</y>
      <width>1661</width>
      <height>841</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="style.qrc">:/new/prefix1/icons/blue2.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="valider">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>510</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 0);</string>
    </property>
    <property name="text">
     <string>valider</string>
    </property>
   </widget>
   <widget class="QPushButton" name="annuler">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>510</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>annuler</string>
    </property>
   </widget>
   <widget class="QComboBox" name="lineEdit_fct">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>410</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Inspecteur</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Technicien</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Receptionniste</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PersonnelAdmin</string>
     </property>
    </item>
   </widget>
   <widget class="QChartView" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>500</y>
      <width>351</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

#refresh:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>500</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/refresh.png</normaloff>:/new/prefix1/icons/refresh.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>tableView</zorder>
   <zorder>form</zorder>
   <zorder>options</zorder>
   <zorder>supprimer</zorder>
   <zorder>modifier</zorder>
   <zorder>lineEdit_nom</zorder>
   <zorder>ajouter</zorder>
   <zorder>lineEdit_pre</zorder>
   <zorder>lineEdit_num</zorder>
   <zorder>lineEdit_cin</zorder>
   <zorder>lineEdit_mail</zorder>
   <zorder>lineEdit_add</zorder>
   <zorder>lineEdit_mdp</zorder>
   <zorder>widget_2</zorder>
   <zorder>valider</zorder>
   <zorder>annuler</zorder>
   <zorder>lineEdit_fct</zorder>
   <zorder>widget_3</zorder>
   <zorder>refresh</zorder>
  </widget>
  <widget class="QWidget" name="side_menu" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>181</width>
     <height>841</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>         Clients</string>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/cli.png</normaloff>:/new/prefix1/icons/cli.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>        Vehicules</string>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/vehi.png</normaloff>:/new/prefix1/icons/vehi.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>  Rendez-Vous</string>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/rend.png</normaloff>:/new/prefix1/icons/rend.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">QtCharts</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="style.qrc"/>
 </resources>
 <connections/>
</ui>
